cpanfile

```diff
@@ -36,6 +36,7 @@ requires 'Mojo::UserAgent';
 requires 'Mojo::Log';
 requires 'Mojolicious::Lite';
 requires 'Net::DBus';
+requires 'Net::Ping';
```

Comment:
please add perl-Net-Ping to the spec file in dist/rpm/os-autoinst.spec as well

Reply:
Fixed.